dandiapi/api/services/version/metadata.py

```diff
@@ -5,7 +5,7 @@
 
 
 def _normalize_version_metadata(
-    raw_version_metadata: dict, embargo: bool, name: str, email: str
+    raw_version_metadata: dict, embargo: bool, name: str, email: str  # noqa: FBT001
```

Comment:
Why exclude this one? Because these arguments (including `embargo`) constitute a uniform and required description of the data to be normalized?

Reply:
@waxlamp Good point. I was just being conservative. Changing this would either require making multiple other arguments keyword-only or changing the order of the arguments. It's an internal API, so it wouldn't break anything to change it, however.

Now they you mentioned it, I'm inclined to change this one too, particularly to have another example of the "right" (really just "safer") way to have boolean arguments.

Reply:
Changed.